packages/buefy-next/src/components/upload/Upload.vue

```diff
@@ -71,44 +80,35 @@ export default {
         }
     },
     emits: ['invalid', 'update:modelValue'],
-    data() {
+    data(): BUploadData {
         return {
-            newValue: this.modelValue,
+            newValue: this.modelValue ?? null,
```

Comment:
You can just leave it as `newValue: this.modelValue`.

Reply:
I believe that I get a TS warning if I do